packages/tools/client-debugger/client-debugger-view/src/ThemeHelper.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { webDarkTheme, webLightTheme, Theme } from "@fluentui/react-components";
+
+/**
+ * Utility function to get the current fluent ui for use
+ * @returns Theme object of FluentUI to be used for dev tool
+ */
+export function flueUIThemeToUse(): Theme {
+	let defaultTheme = webLightTheme;
+
+	if (window.matchMedia?.("(prefers-color-scheme: dark)").matches) {
```

Comment:
Is there documentation we can point to for this query?

Reply:
Added the reference link!